src/app/auth/features/confirm-totp/confirm-totp.component.scss

```diff
@@ -0,0 +1 @@
+// TODO: remove if not needed
```

Comment:
Just delete it. Can be remade later 👍

Reply:
Same for the others

Reply:
Removed now. Generally, we should rarely need individual css thanks to tailwind.